tests/network/setup_multimachine.pm

```diff
@@ -37,6 +43,11 @@ sub run {
     assert_script_run "hostnamectl set-hostname $hostname";
     assert_script_run "hostnamectl status|grep $hostname";
     assert_script_run "hostname|grep $hostname";
+
+    assert_script_run "timedatectl set-ntp yes";
```

Comment:
Why ? tests are booting same image, the time is also same

Reply:
True, this is probably not necessary - this was just a try when I was debugging the salt problem.